back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/idea_plaintext_parser_service.rb

```diff
@@ -2,24 +2,16 @@
 
 module BulkImportIdeas
   class IdeaPlaintextParserService
-    QUESTION_TYPES = %w[select multiselect text text_multiloc multiline_text html_multiloc]
```

Comment:
Have removed this line as there is already a function in custom fields service that removes structural types